gas.md

```diff
@@ -380,7 +380,10 @@ We assume that all operations interrogating the local state have complexity
   the call site.
   ```hs
   computationCost(LOCALCALL(_, nARGS, _) _ rARGS) =
-    saveContextCost + wordCopyCost * (sum [registerSize r | r <- rARGS]) + jumpCost
+    saveContextCost + wordCopyCost * (sum [registerSize r | r <- rARGS])
+    + jumpCost + callStackDepthCheckCost
+  requiredRegisterMemory(LOCALCALL(_, nARGS, _) _ rARGS) =
+    currentRegisterMemory + sum [registerSize(r) | r <- rVALUES]
```

Comment:
1. The memory cost needs to include all the space to save the caller's registers.

2. rVALUES is not defined. Should it be rARGS like in the computation cost.

Reply:
Done. Could you take another look?